src/outputs/textconfig.jl

```diff
@@ -39,7 +39,7 @@ function TextConfig(;
     TextConfig(face, namepixels, namepos, timepixels, timepos, fcolor, bcolor)
 end
 
-function autofont()
+function _autofont()
```

Comment:
Probably `_autofont` could be moved directly under `autofont` for clarity here.

Reply:
Done with 02e5691588135d119bcad971c1cb3cc8f45dcbfc